File: examples/workspaces/booksandrecords-spring-boot/src/main/java/com/ms/booksandrecords/service/dealevent/InvalidQuantity.java
package com.ms.booksandrecords.service.dealevent;

import com.ms.booksandrecords.service.DealEvent;

public class InvalidQuantity extends DealEvent {
    private String id;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }

    private int quantity;

    public InvalidQuantity(String id, int quantity) {
        this.id = id;
        this.quantity = quantity;
    }
}

File: examples/AppModel/booksandrecords-spring-boot/src/main/scala/com/ms/booksandrecords/service/dealevent/InvalidQuantity.scala
package com.ms.booksandrecords.service.dealevent

import com.ms.booksandrecords.service.DealEvent

class InvalidQuantity(var id: String, var quantity: Int) extends DealEvent {
  def getId = id

  def setId(id: String) = this.id = id

  def getQuantity = quantity

  def setQuantity(quantity: Int) = this.quantity = quantity
}